docker-compose.override.yml

```diff
@@ -118,12 +118,11 @@ services:
     environment:
       - REDUX_LOG
 
-  registration-react:
-    build: ./applications/registration-react
+  dataset-catalogue-gui:
     ports:
       - "4300:8080"
     volumes:
-      - ./applications/registration-react/src:/usr/src/app/src
+      - ../dataset-catalogue-gui/src:/usr/src/app/src
```

Comment:
ser ut som du har et punktum for mye her?

Reply:
Fint at dere oppdaterer package.json :)